Measure @Smell annotations that span lines

akowpak-miovision opened the pull request:
We're running sonar 5.1.2 and noticed that our even though the smell rules were detecting the annotation, nothing was showing up as metrics.  We realized it's because our @Smell annotation is normally spread across 3 lines in order to not violate coding standards.  This change will measure @Smell annotations that span lines.

pawlakm commented:
Thanks for your PR, I'll try to review it this WE (the tests are failing and I do not get why as your contribution is not supposed to break anything)

Best regards,

akowpak-miovision commented:
Sorry... I rushed the change and didn't think about tests.  I'm not going to have much time to look at this at the office, but, let me get to the bottom of the tests failing before you spend any time reviewing.

akowpak-miovision commented:
Figured it out... silly mistake.  I have fixed locally and added some new tests.  I'll update the PR at some point today.  Thanks for pointing out the failures.

pawlakm commented:
Thanks for your contribution !
